docs/getting-started.md

```diff
@@ -172,6 +173,34 @@ aepp.configure(
 
 **NOTE** The `environment` parameter is optional and defaults to "prod".
 
+### SSL certificate verification
+
+By default `aepp` will enforce SSL certificate verification on any outgoing HTTP requests.
+
+There can be cases where you might want to disable this verification. This includes, but not limited to:
+- Development workloads might not need the same strong guarantees you would want in production workloads.
+- If you are behind a proxy / VPN you might under certain scenarios get a `SSLCertVerificationError` when trying to connect to AEP APIs.
+
+If you are in one of these situation where disabling SSL certificate verification is appropriate, you can tell `aepp` to do that by using:
```

Comment:
It's a good question, but I would expect in most cases if someone is behind a proxy or on a VPN with special rules they will be aware of it. From what I could see there's only very few situations where SSL certificate verification can fail. If you know other cases let me know I can add more, otherwise we can stick to that for now.